Web/FormHandler.java

```diff
@@ -0,0 +1,141 @@
+		userObject.setcPassword(request.getParameter("cPassword").trim());
+		userObject.setEmail(request.getParameter("email").trim());
+		userObject.setPhoneNumber(request.getParameter("pNumber").trim());
+
+		if (!verifyData(userObject)) {
+		    output.println(errorMessage);
+		} else {
+		    FormHandler.profileHandler(request, response, userObject);
+		}
+	    } else if (request.getParameter("login") != null) {
+		userObject.setName(request.getParameter("user").trim());
+		userObject.setPassword(request.getParameter("password").trim());
+
+		if (!verifyData(userObject)) {
+		    output.println(errorMessage);
+		} else {
+
+		    FormHandler.profileHandler(request, response, userObject);
+		}
+	    } else {
+		response.sendRedirect("index.jsp");
+	    }
+	} catch (NullPointerException | SQLException exception) {
+	    output.println(exception);
+	}
+    }
+
+    private static void profileHandler(HttpServletRequest request, HttpServletResponse response, User userObject) throws SQLException, IOException {
+
+	String query = "";
+	Connection connectionObject = ConnectionUtil.createConnection(response);
+	Statement statementObject = connectionObject.createStatement();
+	PrintWriter output = response.getWriter();
+	query = "SELECT * FROM userAccount WHERE name='" + userObject.getName() + "'";
+	ResultSet result = statementObject.executeQuery(query);
+	HttpSession session = request.getSession();
+
+	if (result.next()) {
+
+	    if (request.getParameter("login") != null) {
+
+		if (result.getString("name").equalsIgnoreCase(userObject.getName()) && result.getString("password").equals(userObject.getPassword())) {
+
+		    session.setAttribute("Username", userObject.getName());
+		    response.sendRedirect("user.jsp");
+		} else {
+
+		    session.setAttribute("loginError", "Either User or Password is incorrect");
+		    response.sendRedirect("index.jsp");
+		}
+	    } else {
+
+		session.setAttribute("message", "User Already Exist");
+		response.sendRedirect("index.jsp");
+	    }
+
+	} else if (request.getParameter("signup") != null) {
+	    query = "INSERT INTO userAccount VALUES('" + userObject.getName() + "','" + userObject.getPassword() + "','" + userObject.getEmail()
+		    + "')";
+	    if (statementObject.executeUpdate(query) != 1) {
+		response.sendRedirect("error.jsp");
+	    } else {
+		output.println("User Created Successfully!!");
+	    }
+	} else {
+	    session.setAttribute("loginError", "Either User or Password is incorrect");
+	    response.sendRedirect("index.jsp");
+	}
+    }
+
+    private boolean verifyData(User userObject) {
+
+	boolean returnValue = true;
+	String name = userObject.getName();
+	String password = userObject.getPassword();
+	String cPassword = userObject.getcPassword();
+	String email = userObject.getEmail();
+	String phoneNumber = userObject.getPhoneNumber();
+	if (name.equals("") || password.equals("")) {
+	    errorMessage = "Form Field can't be empty";
+	    returnValue = false;
```

Comment:
Return value, when you get the condition false first time.Or need to append error Message and throw them all.